test/langtools/jdk/javadoc/doclet/testUnnamedPackage/src1/pkg/package.html

```diff
@@ -0,0 +1,5 @@
+<html>
+    <body>
+        This is a package comment for package pkg.
+    </body>
+</html>
```

Comment:
Old-style?   How quaint!

Reply:
One adapts to the environment... Also, not much missing for valid html5 except `<!doctype html>`